src/components/Landing/styles.scss

```
@@ -0,0 +1,75 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	background-color: #FF60D7;
+	font-size: 1.25rem;
+	line-height: 1;
+}
+
+	.heading, .subheading {
+		font-family: var(--font_family__heading);
+	}
+
+	.heading {
+		margin-bottom: 0.1em;
```

```suggestion
		margin-bottom: 0.6em;
```
I'm getting `36px` here, which would be `0.6em` (also 2em of the body, which is neat)